README.md

```diff
@@ -30,71 +32,130 @@ Scrum Master | [@merridy](https://github.com/merridy)
 Product Owner | [@workbygrant](https://github.com/workbygrant)
 
 ## Comms
+
 Slack: LabPlus-team #feijoa
 
 ## Setup
 
+### Prerequisites
+
+- Postgres
+- Java
```

Comment:
Java??

Reply:
Yup, https://tecadmin.net/setup-elasticsearch-on-ubuntu/
I needed to install it for my "fresh" machine.